### PR TITLE
[REVIEW] Update CONTRIBUTING.md and README.md

### PR DESCRIPTION
**Pull request by Salonijain27**
Updated CONTRIBUTING.md to include the steps to build cuGraph from source.
Updated README.md to include an example code and removed steps to build the cuGraph library from source

### CODE REVIEW

**Comment by afender** [comment]
@Salonijain27 cugraph requires `changelog.md` to be manually updated for each PR.

**Comment by BradReesWork** [comment]
rerun test

File: README.md [hunk 1]
```diff
@@ -4,37 +4,54 @@
 
 The [RAPIDS](https://rapids.ai) cuGraph library is a collection of graph analytics that process data found in GPU Dataframes - see [cuDF](https://github.com/rapidsai/cudf).  cuGraph aims to provide a NetworkX-like API that will be familiar to data scientists, so they can now build GPU-accelerated workflows more easily.
 
- For more project details, see [rapids.ai](https://rapids.ai/).
-
-**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/cudf/blob/master/README.md) ensure you are on the `master` branch.
-
-
-
+For example, the following snippet downloads a CSV file containing the friendship information of students in a karate dojo, it uses the GPU to parse the information into rows and columns and creates a graph. This graph is used to run cuGraph's pagerank algorithm to find the student/students who are the most popular.
```
[the remark this answers is not in the document]
Comment (BradReesWork) [reply README.md]: Why is this in the README versus just a link to the example notebooks.  The example is on how to run PageRank and nothing about why to do graph analytics.  With the Zachary dataset there is no no initial edge weight and is really a bad example of PageRank.

**Comment by BradReesWork** [comment]
Let's drop the example for now.  We will work on a better overall graph example in 0.8

**Comment by Salonijain27** [comment]
removed the example code and edited the link to point to the notebooks in the core notebooks repository and not notebooks-extended

**Comment by afender** [comment]
rerun tests

**Pull request merged**